tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -861,6 +861,46 @@ def _destroy_resource(self):
           handle, ignore_lookup_error=True)
 
 
+def print_row(fields, positions, print_fn):
+  line = ""
+  for i, field in enumerate(fields):
+    field = str(field)
+    end_line_pos = positions[i]
+    if i > 0:
+      line = line + " "
+    line = "{0:{min_length}}".format(line + field, min_length=end_line_pos)
+
+    if len(line) > end_line_pos:
+      line = line[:(end_line_pos - 4)] + " ..."
+
+  print_fn(line)
+
+
+def get_nodes_in_op(graphdef, node_name):
+  ops_in_engine = collections.defaultdict(int)
+  for func in graphdef.library.function:
+    if f"{node_name}_native_segment" == func.signature.name:
+      node_count = len(func.node_def)
+      for node in func.node_def:
+        ops_in_engine[node.op] += 1
+      break
+  return node_count, ops_in_engine
+
+
+def extract_shape(_node, key):
+  out_shape = []
+  for shape in _node.attr[key].list.shape:
+    out_shape.append([dim.size for dim in shape.dim])
+  return out_shape
+
+
+def get_engine_dtype(_node, key):
```

Comment:
s/_node/node/ to follow naming convention

Reply:
Fixed